setup.py

```diff
@@ -12,7 +12,7 @@
     history = history_file.read()
 
 dependency_links = ['git+https://github.com/ryan-rs/qtest-swagger-client.git@master#egg=swagger-client-1.0.0']
-requirements = ['Click>=6.0', 'lxml', 'swagger-client', 'pytest-rpc<1.0.0', 'requests']
+requirements = ['Click>=6.0', 'lxml', 'rpc-qtest-swagger-client<10.0.0', 'pytest-rpc<1.0.0', 'requests']
```

Comment:
Version 10???

Reply:
@johnduarte Yeah, the actual version number of `rpc-qtest-swagger-client` is [9.1.6](https://github.com/rcbops/qtest-swagger-client/tree/v9.1.6). I figured that this was a reasonable constraint so that we don't have to constantly come back and update the constraint.